doc/classes/ConeTwistJoint.xml

```diff
@@ -13,23 +13,23 @@
 	<methods>
 	</methods>
 	<members>
-		<member name="bias" type="float" setter="set_param" getter="get_param">
+		<member name="bias" type="float" setter="set_param" getter="get_param" default="0.3">
 			The speed with which the swing or twist will take place.
 			The higher, the faster.
 		</member>
-		<member name="relaxation" type="float" setter="set_param" getter="get_param">
+		<member name="relaxation" type="float" setter="set_param" getter="get_param" default="1.0">
 			Defines, how fast the swing- and twist-speed-difference on both sides gets synced.
 		</member>
-		<member name="softness" type="float" setter="set_param" getter="get_param">
+		<member name="softness" type="float" setter="set_param" getter="get_param" default="0.8">
 			The ease with which the joint starts to twist. If it's too low, it takes more force to start twisting the joint.
 		</member>
-		<member name="swing_span" type="float" setter="_set_swing_span" getter="_get_swing_span">
+		<member name="swing_span" type="float" setter="_set_swing_span" getter="_get_swing_span" default="45.0">
```

Comment:
For this and `twist_span`, the default value was given in radians. Should be checked.

Reply:
The getter and setter automatically convert to/from radians though https://github.com/godotengine/godot/blob/cd423f47128003d03b94b85a89753bb97e38870c/scene/3d/physics_joint.cpp#L513-L521

Reply:
Ok, so the default value should be correct and the docs were wrong.